PUBLISHING.md

```diff
@@ -0,0 +1,393 @@
+If you're just running a few hosts, or don't want to maintain a fork, then it's easier to change settings at run time.
+
+To change your repo URLs at build time, you would change the `settings.updates.targets-base-url` and `metadata.settings.updates.metadata-base-url.template` settings in [sources/models/defaults.toml](sources/models/defaults.toml).
+This file contains the default settings that will be applied to the host at startup, meaning any host you run already knows to look at your repo.
+(You'll probably want to commit your changes into your fork of the repo; we're working on ways of making it easier to maintain your own model and settings without a fork.)
+
+The easiest way to change your repo URLs at run time is to include the settings changes in user data.
+This method is covered [in README](README.md#using-user-data).
+For example, if you built the `aws-k8s-1.17` variant for `x86_64` and uploaded to the public S3 bucket `my-bucket`, your URLs could look like:
+
+```toml
+[settings.updates]
+targets-base-url = "https://my-bucket.s3-us-west-2.amazonaws.com/targets/"
+metadata-base-url = "https://my-bucket.s3-us-west-2.amazonaws.com/aws-k8s-1.17/x86_64/"
+```
+
+### Waves
+
+When you release a new version, you may want to make your update available to a small number of hosts in the beginning, then gradually expand.
+This can help mitigate the risk of the change and give you more time to detect issues before they're widespread.
+
+The Bottlerocket update system uses the concept of 'waves' of updates.
+For example, you can say that you want:
+* one hour before updates start, so you can prepare
+* 1% of hosts to get the update within 4 hours
+* 5% of hosts to get the update within 1 day
+* 15% of hosts to get the update within 2 days
+* 40% of hosts to get the update within 4 days
+* 60% of hosts to get the update within 5 days
+* 90% of hosts to get the update within 6 days
+* 100% of hosts to get the update after 6 days
+
+This provides a gradual ramp-up so you can watch the status of your deployment more easily.
+And, in fact, this is the default wave policy!
+
+The policy above is defined in [default-waves](sources/updater/waves/default-waves.toml).
+There's also an [accelerated schedule](sources/updater/waves/accelerated-waves.toml) for more urgent deployments, and an ["oh no" schedule](sources/updater/waves/ohno.toml) for emergencies.
+
+If you want to use a different policy, pass `-e PUBLISH_WAVE_POLICY_PATH=sources/updater/waves/chosen-policy.toml` when building your repo.
+For example, to use the accelerated schedule:
+
+```shell
+cargo make repo -e PUBLISH_WAVE_POLICY_PATH=sources/updater/waves/accelerated-waves.toml
+```
+
+To learn more about waves, check out the [README](sources/updater/waves).
+
+### Expiration policy
+
+Each piece of signed metadata in a TUF repo expires after a specific length of time, meaning that repos need to resigned regularly.
```

Comment:
```suggestion
Each piece of signed metadata in a TUF repo expires after a specific length of time, meaning that repos need to re-signed regularly.
```

("resigned" is a word that means something different from what we want here)

Reply:
There's no hard rule, but the general guideline ([1](http://www.gatehousenewsroom.com/2017/08/17/ap-style-guidelines-for-hyphen-usage/) [2](https://blog.apastyle.org/apastyle/2016/10/hyphenation-station-the-hyphenation-of-prefixes-in-apa-style.html)) is to use a hyphen if the prefixed form (`re signed`) could be misunderstood as another word (past tense of `resign`).  I don't think anyone expects that a repo will resign from its job.  Particularly when we're in a paragraph about signing.  And you favor the unhyphenated where possible, per [AP](http://www.gatehousenewsroom.com/2017/08/17/ap-style-guidelines-for-hyphen-usage/).  :P

Reply:
Chicago style could go either way. :P

![hyphen1.jpg](https://user-images.githubusercontent.com/13994/95148824-2fb73c00-0739-11eb-9f38-91bdac078dea.jpg)

![hyphen2.jpg](https://user-images.githubusercontent.com/13994/95148836-3645b380-0739-11eb-9d33-1d5a32e3396a.jpg)

Reply:
(happy to change it, just having fun)

Reply:
English is hard